packages/ui-toolkit/stories/Toaster.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+
+import { StoryFn } from '@storybook/react';
+
+import { Toaster, toast, Button } from '../src/components/atoms';
+import { createPortal } from 'react-dom';
+
+export default {
+  title: 'Toaster',
+  component: toast,
+  tags: [ 'autodocs' ]
+};
+
+
+const Template: StoryFn = (_args) => {
+  const activateToast = (type: string) => {
+    switch (type) {
+      case 'sucess':
+        return toast.success({
+          title: 'Sucess',
+          description: 'Toast Sucess',
+          duration: 5000,
+          dismissible: true,
+          onDismiss: () => console.log('Dismissed'),
+          onAutoClose: () => console.log('Auto Close')
+        });
+
+      case 'warning':
+        return toast.warning({
+          description: 'Toast Warn'
+        });
+
+      case 'error':
+        return toast.error({
+          description: 'Toast error'
+        });
+
+      case 'info':
+        return toast.info({
+          description: 'Toast info'
+        });
+
+      default:
+        toast({ description: 'Default Toast' });
+        return;
+    }
+  };
+
+  const domNode = document.body;
+
+  return (
+    <div className="valign-wrapper width100"
+      style={{ columnGap: '20px' }}
+    >
+      {/* <Toaster /> */}
+      <Button
```

Comment:
Instead of `<Button/>` for the type of Toasts, we can provide a select dropdown with `variants` prop and then the user can select the variant accordingly.

Reply:
added as controls